experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -0,0 +1,648 @@
+)
+
+# Read all site-specific parameters from the parameter file for the site
+include(
+    joinpath(
+        climalsm_dir,
+        "experiments/integrated/fluxnet/$site_ID/$(site_ID)_parameters.jl",
+    ),
+)
+
+# This reads in the data from the flux tower site and creates
+# the atmospheric and radiative driver structs for the model
+include(
+    joinpath(
+        climalsm_dir,
+        "experiments/integrated/fluxnet/met_drivers_FLUXNET.jl",
+    ),
+)
+include(
+    joinpath(
+        climalsm_dir,
+        "experiments/integrated/fluxnet/fluxnet_simulation.jl",
+    ),
+)
+
+# Now we set up the model. For the soil model, we pick
+# a model type and model args:
+soil_domain = land_domain
+soil_ps = Soil.EnergyHydrologyParameters{FT}(;
+    κ_dry = κ_dry,
+    κ_sat_frozen = κ_sat_frozen,
+    κ_sat_unfrozen = κ_sat_unfrozen,
+    ρc_ds = ρc_ds,
+    ν = soil_ν,
+    ν_ss_om = ν_ss_om,
+    ν_ss_quartz = ν_ss_quartz,
+    ν_ss_gravel = ν_ss_gravel,
+    hydrology_cm = vanGenuchten(; α = soil_vg_α, n = soil_vg_n),
+    K_sat = soil_K_sat,
+    S_s = soil_S_s,
+    θ_r = θ_r,
+    earth_param_set = earth_param_set,
+    z_0m = z_0m_soil,
+    z_0b = z_0b_soil,
+    emissivity = soil_ϵ,
+    PAR_albedo = soil_α_PAR,
+    NIR_albedo = soil_α_NIR,
+);
+
+soil_args = (domain = soil_domain, parameters = soil_ps)
+soil_model_type = Soil.EnergyHydrology{FT}
+
+# Soil microbes model
+soilco2_type = Soil.Biogeochemistry.SoilCO2Model{FT}
+
+soilco2_ps = SoilCO2ModelParameters{FT}(;
+    ν = soil_ν, # same as soil
+    θ_a100 = θ_a100,
+    D_ref = D_ref,
+    b = b,
+    D_liq = D_liq,
+    # DAMM
+    α_sx = α_sx,
+    Ea_sx = Ea_sx,
+    kM_sx = kM_sx,
+    kM_o2 = kM_o2,
+    O2_a = O2_a,
+    D_oa = D_oa,
+    p_sx = p_sx,
+    earth_param_set = earth_param_set,
+)
+
+# soil microbes args
+Csom = (z, t) -> eltype(z)(5.0)
+
+soilco2_top_bc = Soil.Biogeochemistry.SoilCO2StateBC((p, t) -> atmos_co2(t))
+soilco2_bot_bc = Soil.Biogeochemistry.SoilCO2FluxBC((p, t) -> 0.0) # no flux
+soilco2_sources = (MicrobeProduction{FT}(),)
+
+soilco2_boundary_conditions =
+    (; top = (CO2 = soilco2_top_bc,), bottom = (CO2 = soilco2_bot_bc,))
+
+soilco2_drivers = Soil.Biogeochemistry.SoilDrivers(
+    Soil.Biogeochemistry.PrognosticMet{FT}(),
+    Soil.Biogeochemistry.PrescribedSOC{FT}(Csom),
+    atmos,
+)
+
+soilco2_args = (;
+    boundary_conditions = soilco2_boundary_conditions,
+    sources = soilco2_sources,
+    domain = soil_domain,
+    parameters = soilco2_ps,
+    drivers = soilco2_drivers,
+)
+
+# Now we set up the canopy model, which we set up by component:
+# Component Types
+canopy_component_types = (;
+    autotrophic_respiration = Canopy.AutotrophicRespirationModel{FT},
+    radiative_transfer = Canopy.TwoStreamModel{FT},
+    photosynthesis = Canopy.FarquharModel{FT},
+    conductance = Canopy.MedlynConductanceModel{FT},
+    hydraulics = Canopy.PlantHydraulicsModel{FT},
```

Comment:
Are we modeling canopy temperature prognostically in these?

Reply:
oops.... we are now

Reply:
:D